--- PKGBUILD
+++ PKGBUILD
@@ -28,14 +28,17 @@ options=(debug)
 _pkgfn="${pkgname/6-/}-everywhere-src-$_qtver"
 source=(https://download.qt.io/official_releases/qt/${pkgver%.*}/$_qtver/submodules/$_pkgfn.tar.xz
         qmake-cflags.patch
-        qmake-config.patch)
+        qmake-config.patch
+        fix-COMPILER_MACROS.patch)
 sha256sums=('d9924d6fd4fa5f8e24458c87f73ef3dfc1e7c9b877a5407c040d89e6736e2634'
             '5411edbe215c24b30448fac69bd0ba7c882f545e8cf05027b2b6e2227abc5e78'
-            '4abc22150fa3e06b2fdcec32146abc9be4e316692aa4d5bd5aa53b4b726783fa')
+            '4abc22150fa3e06b2fdcec32146abc9be4e316692aa4d5bd5aa53b4b726783fa'
+            '93907c3cb050b206390010f9aafc208841329e47a60cee09722e3f2579246a40')
 
 prepare() {
   patch -d $_pkgfn -p1 < qmake-cflags.patch # Use system CFLAGS
   patch -d $_pkgfn -p1 < qmake-config.patch # Don't strip binaries with qmake and use -ltcg, cf. QTBUG-73834
+  patch -d $_pkgfn -p1 < fix-COMPILER_MACROS.patch # Fix: Variable QMAKE_CXX.COMPILER_MACROS is not defined
 }
 
 build() {
