--- PKGBUILD
+++ PKGBUILD
@@ -13,6 +13,12 @@ makedepends=(python)
 source=(rocksdb-$pkgver.tar.gz::https://github.com/facebook/rocksdb/archive/v$pkgver.tar.gz)
 sha256sums=('51a44b8f151d1a39793a8cf37d9de248b980f9aaed0c7bbeee06304e4d7b8d95')
 
+prepare() {
+  cd rocksdb-$pkgver
+  # the next line should be removed as soon as our dtbs can return rv64gc instead of rv64imafdc as isa string
+  sed -i 's/-march=${RISC_ISA}/-march=${RISC_ISA} -misa-spec=2.2/g' build_tools/build_detect_platform
+}
+
 build() {
   cd rocksdb-$pkgver
   PORTABLE=1 USE_RTTI=1 make shared_lib
